cocos/audio/audio-source.ts

```diff
@@ -38,6 +38,11 @@ enum AudioSourceEventType {
     ENDED = 'ended',
 }
 
+class OperationInfo {
```

Comment:
where we used this class, can we use the `OperationInfo ` exported from `operation-queue.ts` ? or should it be an interface ?

Reply:
Used by _operationsBeforeLoading

Reply:
AudioSource only needs two variables, a function name and a parameter; it does not require as many parameter requirements as operation-queue.

Reply:
could it be an interface instead of a class